src/applications/personalization/profile-2/components/ProfileInfoTable.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ProfileInfoTable = ({ data, dataTransformer, fieldName, title }) => (
+  <table className="profile-info-table" data-field-name={fieldName}>
+    <thead>
+      <tr>
+        <th>
+          <h3>{title}</h3>
+        </th>
+      </tr>
```

Comment:
Sorry for the really late comment! I noticed that the `thead` row only has one column while the `tbody` rows have two columns.